apps/api-documenter/src/nodes/DocHeading.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import {
+  IDocNodeParameters,
+  DocNode
+} from '@microsoft/tsdoc';
+import { CustomDocNodeKind } from './CustomDocNodeKind';
+
+/**
+ * Constructor parameters for {@link DocHeading}.
+ */
+export interface IDocHeadingParameters extends IDocNodeParameters {
+  title: string;
+  level?: number;
+}
+
+/**
+ * Represents table, similar to an HTML `<h1>` or `<h2>` element.
```

Comment:
>table [](start = 14, length = 5)

Should this be "Represents header" or something similar?

Reply:
Yup.  Also the sentence is missing some words.

---
In reply to: [236556052](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236556052) [](ancestors = 236556052)